.gitignore

```diff
@@ -87,3 +87,4 @@ ENV/
 
 # Rope project settings
 .ropeproject
+.DS_Store
```

Comment:
This should be deleted and gitignored from every folder.

Please make sure it doesn't come through to the PR.